Tasks/Common/azure-blobstorage-artifactProvider-v2/azureBlobStorageProvider.ts

```diff
@@ -73,6 +76,10 @@ export class AzureBlobProvider implements models.IArtifactProvider {
             } else {
                 readStream = this._blobSvc.createReadStream(this._container, artifactItem.path, null);
             }
+            const fileName: string = this._getFileName(artifactItem.path);
+            if (fileName) {
+                artifactItem.path = fileName;
```

Comment:
artifactItem.path = this._getFileName(artifactItem.path) || artifactItem.path
As an idea for shortening

Reply:
@AlexandrKravchuk these lines are removed in accordance with your previous review point.